configure.py

```diff
@@ -807,52 +807,6 @@ def set_other_cuda_vars(environ_cp):
       write_to_bazelrc('test --config=cuda')
 
 
-def set_host_cxx_compiler(environ_cp):
```

Comment:
Is this still needed for triSYCL? It looks like the HOST_CXX_COMPILER variable is still being used in https://github.com/lukeiwanski/tensorflow/blob/dev/eigen_mehdi/third_party/sycl/sycl_configure.bzl#L238